lib/pbench/test/unit/server/test_requests.py

```diff
@@ -77,97 +100,130 @@ def test_json_object(client, caplog, server_config):
 
 class TestUpload:
     @staticmethod
-    def test_missing_filename_header_upload(client, caplog, server_config):
-        expected_message = (
-            "Missing filename header, "
-            "POST operation requires a filename header to name the uploaded file"
-        )
+    def test_missing_authorization_header(client, caplog, server_config):
```

Comment:
Should we throw in a bad authorization header, too? In theory that's tested elsewhere, and the test case you added ensures that PUT has the decorator, so that may be redundant. Are redundant security tests bad? ;-)

Reply:
Tracked in #2246.